iot_input/models/iot_device.py

```diff
@@ -30,3 +34,89 @@ def action_show_input(self):
             result['views'] = [(False, 'form')]
             result['res_id'] = self.input_ids.id
         return result
+
+    @api.multi
+    def parse_single_input(self, value):
+        '''Handle single input for device
+
+        :param dict value:
+            Dict containing at least keys 'address', 'value'
+        :returns: dict with keys 'status', 'message' where:
+            - status='ok' when value is parsed without errors
+            - status='error' and message='error message' when error occurs
+            If value contains a value with key 'uuid', it is passed in the return dict
+            to identify result for each entry at the iot end
+        :rtype: dict
+        '''
+        if 'address' not in value.keys():
+            _logger.warning('Adress for Input %s is required', value['address'])
```

Comment:
address is not in value, so, this should fail :thinking:

Reply:
Yes, caught that just a few minutes ago. The same goes a couple of lines below with value.